lang/en.json

```diff
@@ -1,17 +1,18 @@
 {
 
-  "dfreds-convenient-effects.conditionAlmostdead": "Almost dead",
-  "dfreds-convenient-effects.conditionBlinded": "Blind",
+  "dfreds-convenient-effects.conditionAlmostDead": "Almost dead",
+  "dfreds-convenient-effects.conditionBlinded": "Blinded",
   "dfreds-convenient-effects.conditionCharmed": "Charmed",
-  "dfreds-convenient-effects.conditionConcentrating": "Concentrating",
+  "dfreds-convenient-effects.conditionConcentration": "Concentration",
```

Comment:
I think this was correct before and should be 'Concentrating'

Reply:
Strange it was correct on my [fork](https://github.com/p4535992/dfreds-convenient-effects/blob/609350280cfd7913bcc31c2be1724f1ec1ce9c7f/lang/en.json#L6), but i'll check out